### PR TITLE
Polkadot-js extension integration

### PR DESCRIPTION
**Pull request by EugenWay**
Resolves # .

Provide functionality to manage Substrate account from Polkadot extension on Gear framework

@reviewer-or-team

### CODE REVIEW

File: website/frontend/src/components/blocks/ProgramDetails/ProgramDetails.tsx [hunk 1]
```diff
@@ -61,6 +62,9 @@ export const ProgramDetails: VFC<Props> = ({ setDroppedFile, droppedFile }) => {
     metaFieldRef.current?.click();
   };
 
+  const currentAccount = useSelector((state: RootState) => state.account.account);
+  console.log(currentAccount);
```
Comment (aderrod94) [inline website/frontend/src/components/blocks/ProgramDetails/ProgramDetails.tsx]: unnecessary console.log

File: website/frontend/src/components/blocks/Wallet/Wallet.tsx [hunk 2]
```diff
@@ -76,18 +71,18 @@ export const Wallet = () => {
...
-  const selectAccount = (event: any, index: number) => {
+  const selectAccount = (event: MouseEvent, index: number) => {
     event.stopPropagation();
     if (ingectedAccounts) {
-      ingectedAccounts.forEach((acc: any, i: any) => {
+      ingectedAccounts.forEach((acc: UserAccount, i: number) => {
         acc.isActive = false;
         if (i === index) {
           acc.isActive = true;
+          console.log(1);
```
Comment (aderrod94) [inline website/frontend/src/components/blocks/Wallet/Wallet.tsx]: unnecessary console.log

File: website/frontend/src/components/blocks/AccountList/AccountList.tsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+import { toShortAddress } from '../../../helpers';
+
+import './AccountList.scss';
+
+type Props = {
+  list: Array<UserAccount>;
+  toggleAccount: (event: any, index: number) => void;
+};
+
+export const AccountList = ({ list, toggleAccount }: Props) => {
```
Comment (codev0) [inline website/frontend/src/components/blocks/AccountList/AccountList.tsx:14]:
```suggestion
export const AccountList: FC<Props> = ({ list, toggleAccount }) => {
```

File: website/frontend/src/components/blocks/AccountList/AccountList.tsx [hunk 4]
```diff
@@ -0,0 +1,42 @@
+import React from 'react';
```
Comment (codev0) [inline website/frontend/src/components/blocks/AccountList/AccountList.tsx:1]:
```suggestion
import React, { FC } from 'react';
```

File: website/frontend/src/components/blocks/AccountList/AccountList.tsx [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+  toggleAccount: (event: any, index: number) => void;
+};
+
+export const AccountList = ({ list, toggleAccount }: Props) => {
+  const accountItem = list.map((account: UserAccount, index: number) => (
+    <button
+      type="button"
+      className={clsx('account-list__item', account.isActive && 'active')}
+      key={account.address}
+      onClick={(event: any) => {
```
Comment (codev0) [inline website/frontend/src/components/blocks/AccountList/AccountList.tsx:20]:
```suggestion
      onClick={(event: React.MouseEvent<HTMLButtonElement>) => {
```

File: website/frontend/src/components/blocks/AccountList/index.tsx [hunk 6]
```diff
@@ -0,0 +1,3 @@
+import { AccountList } from './AccountList';
```
Comment (codev0) [inline website/frontend/src/components/blocks/AccountList/index.tsx:1]:
```suggestion
export { AccountList } from './AccountList';
```

File: website/frontend/src/components/blocks/Wallet/Wallet.tsx [hunk 7]
```diff
@@ -1,31 +1,134 @@
...
+      getBalance(currentAccount.address);
+    }
+  }, [currentAccount, api, dispatch]);
+
+  const toggleModal = () => {
+    setIsOpen(!isOpen);
+  };
+
+  useEffect(() => {
+    if (isOpen) document.body.style.overflow = 'hidden';
```
Comment (codev0) [inline website/frontend/src/components/blocks/Wallet/Wallet.tsx:67]: we could try to use this one https://github.com/willmcpo/body-scroll-lock#reactes6

**Review by codev0 — APPROVED** [review]

**Review by osipov-mit — APPROVED** [review]

**Pull request merged**